core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -160,6 +160,8 @@
      */
     public static final String ID_DEFAULT = SystemProperties.getString(UpdateCenter.class.getName()+".defaultUpdateSiteId", "default");
 
+    public static final String LEGACY_ID_DEFAULT = "default";
```

Comment:
"legacy" does not seem to be a right wording, because it points to the actual site.